ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -114,6 +114,11 @@ function handleAnyDateFilter(propertyValueMap, filter) {
   )
   existingFilter.type = filter.filters ? filter.filters[0].type : filter.type
   existingFilter.value = filter.filters ? filter.filters[0].value : filter.value
+
+  if (existingFilter.type === 'DURING') {
```

Comment:
`cql.js` only cares about `from` and `to` for DURING filters. The query date views seem to assume that every filter uses a `value` field, which `DURING` filters do not have. That is why I have to do all this conversion between `from`/`to` and `value`.

It seems like we could modify `cql.js` such that it fills out the `value` field in addition to the `from` and `to` fields for DURING filters and simplify much of this code.

Thoughts? @codice/ui

Reply:
I think it would be worth trying to update those views to build the filter how cql expects.